.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-5.0.0-beta3
\ No newline at end of file
+6.6.0
```

Comment:
hopefully the generator upgrade is backwards compatible

Reply:
i can downgrade to keep it consistent, might be ok though?